spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowSuspiciousConnectsModel.scala

```diff
@@ -133,23 +121,38 @@ class FlowSuspiciousConnectsModel(topicCount: Int,
 object FlowSuspiciousConnectsModel {
 
   val ModelSchema = StructType(List(HourField,
-    MinuteField,
-    SecondField,
     SourceIPField,
     DestinationIPField,
     SourcePortField,
     DestinationPortField,
-    IpktField,
-    IbytField))
+    ProtocolField,
+    IbytField,
+    IpktField))
 
   val ModelColumns = ModelSchema.fieldNames.toList.map(col)
 
-  def trainNewModel(sparkContext: SparkContext,
-                    sqlContext: SQLContext,
-                    logger: Logger,
-                    config: SuspiciousConnectsConfig,
-                    inputRecords: DataFrame,
-                    topicCount: Int): FlowSuspiciousConnectsModel = {
+
+  def cleanData(flows: DataFrame): DataFrame = {
```

Comment:
we can move the change to SPOT-128 if you like, but I think that the cleaning code should be packaged with the model because it is enforcing preconditions necessary to build the model